In the upstream desktop file, use the icon name of what we install (Closes: #713808).
---
 misc/minetest.desktop |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/misc/minetest.desktop
===================================================================
--- a/misc/minetest.desktop
+++ b/misc/minetest.desktop
@@ -3,7 +3,7 @@ Name=Minetest
 GenericName=Minetest
 Comment=Multiplayer infinite-world block sandbox
 Exec=minetest
-Icon=minetest-icon
+Icon=minetest
 Terminal=false
 Type=Application
 Categories=Game;
